### PR TITLE
(HYD-110) Add pypi description

### PR DESCRIPTION
**Pull request by trthhrtz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Hydrosphere Serving SDK
 
-This package contains python API for hydroserving. 
+This package contains a python API for hydroserving.
```
Comment (KineticCookie) [inline README.md:3]:
```suggestion
The package contains an implementation of Hydroserving API.
Tested on python 3.6, 3.7, 3.8.
```

File: setup.py [hunk 2]
```diff
@@ -1,5 +1,11 @@
+from os import path
+
 from setuptools import setup, find_packages
 
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
```
Comment (KineticCookie) [inline setup.py:6]: Why do you need absolute path here?
```suggestion
with open('README.md', encoding='utf-8') as f:
```

Comment (trthhrtz) [reply setup.py:6]: Was following official guide (https://packaging.python.org/guides/making-a-pypi-friendly-readme/)

**Comment by trthhrtz** [comment]
@KineticCookie Regarding the abspath, was following the official pypi guide (https://packaging.python.org/guides/making-a-pypi-friendly-readme/)

**Pull request merged**